cmd/renterd/main.go

```diff
@@ -44,6 +44,22 @@ const (
 	// minutes. That's why we assume 10 seconds to be more than frequent enough
 	// to refill an account when it's due for another refill.
 	defaultAccountRefillInterval = 10 * time.Second
+
+	// usageHeader is the header for the CLI usage text.
+	usageHeader = `
+Renterd is the official Sia renter daemon. It provides a REST API for forming
+contracts with hosts, uploading data to them and downloading data from them.
+
+There are 3 ways to configure renterd:
```

Comment:
`nit` if you re-order it here and go `yaml`, `flags`, `env` we can add the context that they're respected in that particular order